lib/ex_metrics.ex

```diff
@@ -13,10 +13,9 @@ defmodule ExMetrics do
 
   defmacro timeframe(key, do: yield) do
     quote do
-      before_time = :os.timestamp()
+      before_time = System.monotonic_time(:microsecond)
       result = unquote(yield)
-      after_time = :os.timestamp()
-      diff = :timer.now_diff(after_time, before_time)
+      diff = (System.monotonic_time(:microsecond) - before_time) |> abs
 
       ExMetrics.timing(unquote(key), diff / 1_000)
```

Comment:
Could also try to use the intended time unit? It seems the following is attempting to provide diff in `ms`. Using `System.monotonic_time(:millisecond)` will provide a `ms` diff directly without needing the 1000 division. This save generating extra immutable data.

```ex
ExMetrics.timing(unquote(key), diff / 1_000)
```

Reply:
Good should, I'll do that 👍